tests/console/zypper_in.pm

```diff
@@ -32,6 +45,20 @@ sub run {
     clear_console;    # clear screen to see that second update does not do any more
     assert_script_run("rpm -e $pkgname");
     assert_script_run("! rpm -q $pkgname");
+
+    script_run 'touch /preinstall_fail';
+    script_run 'zypper -n in --allow-unsigned-rpm ' . data_url('zypper/pa.rpm');
+    script_run 'rm -f /preinstall_fail';
+
+    assert_script_run 'export ZYPP_SINGLE_RPMTRANS=1';
+
+    script_run 'touch /preinstall_fail';
+    script_run 'zypper -n in --allow-unsigned-rpm ' . data_url('zypper/pa.rpm');
+    script_run 'rm -f /preinstall_fail';
+
+    assert_script_run 'zypper -n in adaptec-firmware adobe-sourcecodepro-fonts alsa-devel apache2 atmel-firmware';
```

Comment:
adobe-sourcecodepro-fonts is a 'problem' on Tumbleweed, especially on staging. The branding has recently been changed to 'adobe-sourcesans3-fonts' (`s/pro/3/`) - for this reason, staging currently can't pass (as the font is not part of the minimal DVD test; on tw with repo, the font still exists, which explains why the VR passed)

See e.g. https://openqa.opensuse.org/tests/2219945#step/zypper_in/27